config/environments/production.rb

```diff
@@ -65,9 +65,15 @@
   # Ignore bad email addresses and do not raise email delivery errors.
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
-
+  config.action_mailer.delivery_method = :mailgun
+  config.action_mailer.mailgun_settings = {
+      api_key: ENV['MAILGUN_API_KEY'],
+      domain: ENV['MAILGUN_DOMAIN'],
+      # api_host: 'api.eu.mailgun.net'  # Uncomment this line for EU region domains
+  }
+  config.action_mailer.perform_deliveries = true
   # TODO: set a proper host
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.default_url_options = { host: 'rideboard.app', port: 3000 }
```

Comment:
For production, the port is 443, the default HTTPS port.  3000 is only used in development.  Just leave out the port option and it will use the default which is what we want.

Hard coding the rideboard.app domain will also make this very hard to test without pushing to production.  However, if it works in development, I'm willing to push this and then find a more general way to set it up when we have more time.